### PR TITLE
New: EuSoft.Eudic version 12.6.4

### PR DESCRIPTION
**Pull request by SpecterShell**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25442)

### CODE REVIEW

**Comment by SpecterShell** [comment]
Both direct download and sandbox test can give me the correct hash. Maybe something went wrong while downloading the installer.🤔

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
from Windows Sandbox:
image

**Comment by SpecterShell** [comment]
> from Windows Sandbox:
> image

from Windows Sandbox as well:
image

File: manifests/e/EuSoft/Eudic/12.6.4/EuSoft.Eudic.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+InstallModes:
+- interactive
+- silentWithProgress
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://static.frdic.com/pkg/eudic_win.exe?v=2021-08-03
+  InstallerSha256: B523E27A26F7EBDA6D9F1A87E7F3CF182C83317265AC9805E59CF3C21DE98F07
```
Comment (ItzLevvie) [inline manifests/e/EuSoft/Eudic/12.6.4/EuSoft.Eudic.installer.yaml:21]:
```suggestion
  InstallerUrl: https://static.frdic.com/pkg/eudic_win.exe
  InstallerSha256: 9b3ad96d5a4b5aef051671d8ca13e13820902aedde86690e7a51e760aa038e9b
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by SpecterShell** [comment]
The publisher just released a new version today but they haven't update all their pages yet. I don't know why it is redirected to the latest version in some cases since I can still get the old version with the same url.🤣
I will close this pull request and make another one later.

**Pull request closed without merging**